### PR TITLE
[CLI] Support session key in CLI

### LINKED ISSUE #589: [CLI] CLI support SessionKey
part of #419 

1. Generate session key via Ed25519 master key.
2. Support sign transactions via Session Key.
3. Add CLI test for the SessionKey.

Maybe we can provide a console for cli, and auto-generate a session key for the console session in the future.

### PR DESCRIPTION
**Pull request by jolestar**
resolve #589 

1. Rename typo ModuleBundle -> ModuleBinding.
2. RPC Client support ModuleBinding call.
3. Simply the KeyStore serialize and deserialize. cc @feliciss 
4. Add command `rooch session-key create`.
5. Add tests for SessionKey.

TODO
1. Implement the scope check for SessionKey. 
2. add `list` and `remove` commands for SessionKey. #630

### CODE REVIEW

File: crates/rooch-framework/sources/session_key.move [hunk 1]
```diff
@@ -107,20 +103,20 @@ module rooch_framework::session_key {
...
-        let auth_key = if(scheme == ed25519_validator::scheme()){
-            ed25519_validator::get_authentication_key_from_authenticator_payload(&authenticator_payload)
-        }else{
-            //TODO support other built-in validators
+        // We only support ed25519 validator for SessionKey now
+        if(scheme != ed25519_validator::scheme()){
             return option::none()
         };
+
+        let auth_key = ed25519_validator::get_authentication_key_from_authenticator_payload(&authenticator_payload);
```
Comment (baichuan3) [inline crates/rooch-framework/sources/session_key.move:111]: Is the method name too long?

File: crates/rooch/src/commands/session_key/mod.rs [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+            }),
+        }
+    }
+}
+
+#[derive(clap::Subcommand)]
+#[clap(name = "session_key")]
+pub enum SessionKeyCommand {
+    Create(CreateCommand),
+}
```
[the remark this answers is not in the document]
Comment (jolestar) [reply crates/rooch/src/commands/session_key/mod.rs:34]: TODO #630

**Review by wow-sven — APPROVED** [review]

File: crates/rooch-framework/sources/session_key.move (hunk 1 above)
Comment (feliciss) [reply crates/rooch-framework/sources/session_key.move:111]: Yes. We can use self description.

Comment (feliciss) [reply crates/rooch-framework/sources/session_key.move:111]: Or import the function directly and use it, eliminating the need to use `<handle>::`.

**Pull request merged**